march_hardware_interface/src/march_hardware_interface_node.cpp

```diff
@@ -58,7 +58,7 @@ int main(int argc, char** argv)
       march.waitForPdo();
 
       const ros::Time now = ros::Time::now();
-      ros::Duration elapsed_time = std::min(now - last_update_time, minimal_elapsed_time);
+      ros::Duration elapsed_time = now - last_update_time;
```

Comment:
Also remove the minimal_elapsed_time variable? (You can just initialize the last_update_time to now)